src/main/java/org/spongepowered/api/conversation/ExternalChatHandlers.java

```diff
@@ -0,0 +1,123 @@
+/*
+ * This file is part of SpongeAPI, licensed under the MIT License (MIT).
+ *
+ * Copyright (c) SpongePowered <https://www.spongepowered.org>
+ * Copyright (c) contributors
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ * THE SOFTWARE.
+ */
+package org.spongepowered.api.conversation;
+
+import org.spongepowered.api.text.Text;
+
+import java.util.ArrayDeque;
+import java.util.Queue;
+
+/**
+ * The main external chat handlers you would need.
+ */
+public class ExternalChatHandlers {
+
+    /**
+     * Creates an external chat handler which lets through all messages so
+     * the {@link Conversant} sees all the messages even within the conversation.
+     *
+     * @return An external chat handler which lets through all messages
+     */
+    public static ExternalChatHandler letThrough() {
+        return new LetThroughExternalChatHandler();
```

Comment:
This is pretty smart, I didn't think of that, but I will only be able to do so for the `letThrough` and `deleteAll` handlers as the other needs to be unique per player.